### PR TITLE
Updating "Getting Started" tutorial to latest versions

### PR DESCRIPTION
**Pull request by akosma**
This PR provides an update to the "Getting Started" tutorial to the latest versions available of various Syn components.

## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [ x Update the documentation.
- [ ] Update the ./CHANGELOG.md.
- [ ] Link this PR to related issues.

### CODE REVIEW

**Comment by tobru** [comment]
I think it would make sense to check the new features from https://github.com/projectsyn/lieutenant-operator/releases/tag/v0.5.0 and see how they could be integrated into to the tutorial to make use of them. @corvus-ch could help you with them.

File: docs/modules/ROOT/pages/tutorials/getting-started.adoc [hunk 1]
```diff
@@ -397,18 +389,15 @@ This should list 5 Pods, maybe still in `ContainerCreating`.
...
 kubectl -n syn get secret steward -o json | jq -r .data.token | base64 --decode
+kubectl -n syn port-forward pod/$(kubectl -n syn get pod -o custom-columns=:metadata.name | grep argocd-server) 8080:8080
 ----
-. Now you can access Argo CD by forwarding the port and opening it in your browser with http://localhost:8443. Login with the username `admin` and the password retrieved in the previous step.
 +
-[source,shell]
-----
-kubectl -n syn port-forward svc/argocd-server 8443:443
-----
+. Now you can access Argo CD by forwarding the port and opening it in your browser with http://localhost:8080. Login with the username `admin` and the password retrieved in the previous step.
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/tutorials/getting-started.adoc:400]: The port forward already happend at this point. All that is left, is opening it in the browser. Or did I miss something?

**Review by corvus-ch — CHANGES_REQUESTED** [review]

**Comment by corvus-ch** [comment]
> I think it would make sense to check the new features from https://github.com/projectsyn/lieutenant-operator/releases/tag/v0.5.0 and see how they could be integrated into to the tutorial to make use of them. @corvus-ch could help you with them.

Agree, but let us do this within a change on its own. I would like to see the docs up to date as fast as possible.

**Comment by tobru** [comment]
> Agree, but let us do this within a change on its own. I would like to see the docs up to date as fast as possible.

Fine be me. Just make sure to not forget to do that later then.

**Comment by corvus-ch** [comment]
Replaced by  #162.

**Pull request closed without merging**